### PR TITLE
[WIP] Using new version of auth0.js an Auth0LockPasswordless

### PR DESCRIPTION
**Pull request by luisrudge**

### CODE REVIEW

File: custom-email-code.html [hunk 1]
```diff
@@ -74,11 +74,11 @@ <h1 id="logo"><img src="auth0_logo_final_blue_RGB.png" /></h1>
       };
```
Comment (aaguiarz) [inline custom-email-code.html]: Should be err.description instead of err.error_description

File: custom-email-code.html [hunk 2]
```diff
@@ -74,11 +74,11 @@ <h1 id="logo"><img src="auth0_logo_final_blue_RGB.png" /></h1>
...
-      function login(){
+      function login() {
         console.log('logging in...');
         var email = $('input.email').val();
         var code = $('input.code').val();
-        webAuth.passwordlessVerify({
+        webAuth.passwordlessLogin({
           connection: 'email',
           email: email,
           verificationCode: code
```
Comment (aaguiarz) [inline custom-email-code.html]: I'm getting that I should enable Cross Origin Authentication. OIDC conformant is not set anywhere, not sure how to turn it off. It also happens with custom-email-sms

File: custom-magic-link.html [hunk 3]
```diff
@@ -70,7 +70,7 @@ <h1 id="logo"><img src="auth0_logo_final_blue_RGB.png" /></h1>
       };
 
       // Check for returned token and parse
```
Comment (aaguiarz) [inline custom-magic-link.html]: Should be err.description

File: custom-sms.html [hunk 4]
```diff
@@ -72,10 +72,10 @@ <h1 id="logo"><img src="auth0_logo_final_blue_RGB.png" /></h1>
       };
```
Comment (aaguiarz) [inline custom-sms.html]: Should be err.description

File: lock-magic-link.html [hunk 5]
```diff
@@ -1,84 +1,76 @@
...
-    <script type="text/javascript">
-
-      // Initialize Passwordless Lock instance
-      var lock = new Auth0LockPasswordless(
-        // All these properties are set in auth0-variables.js
-        AUTH0_CLIENT_ID,
-        AUTH0_DOMAIN
-      );
+  </div>
+  <script src="https://321-22887863-gh.circle-artifacts.com/0/home/ubuntu/lock/build/lock.min.js"></script>
```
Comment (luisrudge) [inline lock-magic-link.html]: Can only do that when we release lock

File: lock-email-code.html [hunk 6]
```diff
@@ -1,89 +1,75 @@
...
+        <button onclick="login()" class="btn btn-primary btn-lg btn-login btn-block">SignIn</button>
+      </div>
+      <div class="logged-in-box auth0-box logged-in" style="display: none;">
+        <h1 id="logo"><img src="auth0_logo_final_blue_RGB.png" /></h1>
+        <img class="avatar" />
+        <h2>Welcome <span class="nickname"></span></h2>
+      </div>
+    </div>
+  </div>
+  <script src="https://321-22887863-gh.circle-artifacts.com/0/home/ubuntu/lock/build/lock.min.js"></script>
```
Comment (luisrudge) [inline lock-email-code.html]: Can only do that when we release lock

File: README.md [hunk 7]
```diff
@@ -28,7 +28,7 @@ You can read more about Passwordless Authentication on our [documentation Site](
 	```
 	> You can get the `{CLIENT_ID}` and `{DOMAIN}` values from the [Auth0 Dashboard](https://manage.auth0.com).
 
-2. In your App's configuration on the [Auth0 Dashboard](https://manage.auth0.com), add `http://localhost:3000` to the **Allowed Origins (CORS)** list in order for the one time code samples to work, and add `http://localhost:3000/custom-magic-link.html, http://localhost:3000/lock-magic-link.html, http://localhost:3000/lock-sms.html, http://localhost:3000/lock-email-code.html` to the **Allowed Callback URLs** list for the magic link samples to function. 
+2. In your App's configuration on the [Auth0 Dashboard](https://manage.auth0.com), add `http://localhost:3000` to the **Allowed Origins (CORS)** list in order for the one time code samples to work, and add `http://localhost:3000/custom-magic-link.html, http://localhost:3000/custom-email-code.html, http://localhost:3000/lock-magic-link.html, http://localhost:3000/lock-sms.html, http://localhost:3000/lock-email-code.html` to the **Allowed Callback URLs** list for the magic link samples to function.
```
Comment (jeffreylees) [inline README.md]: In your Client settings in the [Dashboard](https://manage.auth0.com), add the following to the **Allowed Origins (CORS)** list in order for the one time code samples to work:

```
http://localhost:3000
```

You will also need to add the following to the **Allowed Callback URLs** list for the magic link samples to function: 

```
http://localhost:3000/custom-magic-link.html, 
http://localhost:3000/custom-email-code.html, 
http://localhost:3000/lock-magic-link.html, 
http://localhost:3000/lock-sms.html, 
http://localhost:3000/lock-email-code.html
```

File: custom-email-code.html (hunk 2 above)
Comment (jeffreylees) [inline custom-email-code.html]: The method `passwordlessLogin` isn't documented (in the docs site, README, or at the generated docs)  - is it good to use? Do we need to document it?

File: lock-email-code.html [hunk 8]
```diff
@@ -1,89 +1,75 @@
...
 <html>
-  <head>
-    <meta charset="utf-8">
-    <script src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
 
-    <script src="//use.typekit.net/iws6ohy.js"></script>
-    <script>try{Typekit.load();}catch(e){}</script>
+<head>
+  <meta charset="utf-8">
+  <script src="http://code.jquery.com/jquery-2.1.1.min.js"></script>
```
Comment (jeffreylees) [inline lock-email-code.html]: Guess we should make that `https`?

File: lock-magic-link.html [hunk 9]
```diff
@@ -1,84 +1,83 @@
...
+
+<body class="home">
+  <div class="container">
+    <div class="login-page clearfix">
+      <div class="login-box auth0-box before">
+        <img class="logo" src="https://i.cloudup.com/StzWWrY34s.png" />
+        <h3>Auth0 Passwordless Example</h3>
+        <p>Login with a Magic Link sent by Email</p>
+        <div class="alert alert-warning">Don't forget to add your page's origin
+          <strong><script type="text/javascript">document.write(location.origin)</script></strong> to your App's <strong>Allowed Origins (CORS)</strong>          and
```
Comment (jeffreylees) [inline lock-magic-link.html]: App -> Client - same in the other below examples too

File: custom-email-code.html (hunk 2 above)
Comment (luisrudge) [inline custom-email-code.html]: It's a new method on auth0.js still needs more info 😊

File: lock-magic-link.html (hunk 9 above)
Comment (luisrudge) [inline lock-magic-link.html]: done

File: lock-email-code.html (hunk 8 above)
Comment (luisrudge) [inline lock-email-code.html]: done

**Pull request merged**